Add description of instance state (#2649)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

In the instances list, any Compute type unregistered instance can be registered (imported) into Cloud Application Manager to enable lifecycle management on it (an icon button allowing it will appear at the end of its row), so it will be shown from them on as a registered instance in the corresponding views. The Unregistered Instances tab in the provider details page remains unchanged showing only compute registerable instances. If you want to bulk register (import) several instances from the same provider, use this feature from there instead, where bulk register is also available as a bulk action.

#### Instance states

> * **Online**: the instance is running and ready for use.

Reviewer: ```suggestion
* **Online**: last lifecycle operation succeeded and the instance is up and running.
```

> * **Unavailable**: the instance actions failed and the instance is not accessible. 

Reviewer: ```suggestion
* **Unavailable**: last lifecycle operation has failed, see instance logs for more details. The instance may not be accessible. 
```

> * **Processing**: when instance actions are being executed. The instance is not running yet.

Reviewer: ```suggestion
* **Processing**: when a lifecycle operation is being executed. The instance is not ready yet.
```

> * **Not Responding**: the instance is not responding to the agent of Cloud Application Manager, during the last ten minutes.

Reviewer: ```suggestion
* **Not Responding**: the Cloud Application Manager instance agent has not contacted back for at least 10 minutes. The instance may have been changed from outside of Cloud Application Manager, and could be stopped, deleted or just the agent process not running.
```

> * **Unknown**: Cloud Application Manager cannnot determinate the state of the instance. The instance has several machines and someones are in the Not Responding state and others in other.

Reviewer: ```suggestion
* **Unknown**: Cloud Application Manager cannot determinate the state of the instance. The instance has several machines and someones are in the Not Responding state and some others in any other state.
```

* **Shutdown**: the instance is shut down and cannot be used. The instance can be restarted at any time.

> * **Terminated**: the instance has been permanently deleted and cannot be restarted.

Reviewer: ```suggestion
* **Terminated**: the instance has been removed from the underlying provider and it is no longer available. Cloud Application Manager keep the instance data and its logs, that will be removed if you *delete* the instance.
```
